source/includes/_data-dictionary.md.erb

```diff
@@ -19,14 +19,16 @@ Asset characteristics are posted per asset, not per reporting year.
 | construction_year    	| integer                 	| <%= dictionary_description(:construction_year) %>    	|
 | asset_ownership    	  | numeric                 	| <%= dictionary_description(:asset_ownership) %>    	  |
 
-Certifications are posted per asset, not per reporting year.
+Certifications and energy ratings are posted per asset, not per reporting year.
 
 | variable         	| type                    	| description                                      	| unit                 	|
 |------------------	|-------------------------	|--------------------------------------------------	|----------------------	|
 | id 			          | integer                 	| <%= dictionary_description(:id) %>		 	          |                      	|
 | certification_id 	| integer                 	| <%= dictionary_description(:certification_id) %> 	|                      	|
 | size             	| numeric                 	| <%= dictionary_description(:size) %>             	| m<sup>2</sup>/sq. ft 	|
 | level            	| character varying (255) 	| <%= dictionary_description(:level) %>            	|                      	|
+| energy_rating_id           | integer                 | <%= dictionary_description(:energy_rating_id) %>           |                      |
+| energy_rating_size         | numeric                 | <%= dictionary_description(:energy_rating_size) %>         | m<sup>2</sup>/sq. ft |
```

Comment:
would be the same as for `certifications`, namely `size`

it's okay to keep one row for both

just got to update the description: "Foor area covered by the certification or energy rating."

which has to be done in https://github.com/GRESB/api-docs/blob/master/data/dictionary.yml#L171